packages/chain-adapters/src/cosmossdk/osmosis/OsmosisChainAdapter.ts

```diff
@@ -269,6 +271,56 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.OsmosisMain
     }
   }
 
+  async buildLPAddTransaction(
```

Comment:
adding cli example calling these functions might be helpful during the dev process later when integrating with web. maybe it doesnt matter tho